content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -942,3 +942,126 @@ a scheduled action, but you can also send each invoice manually if needed.
 
 Once the invoice is validated with |SUNAT|, customers can download the :file:`.zip` with the CDR,
 XML, and PDF files directly from the customer portal by clicking the :guilabel:`Download` button.
+
+Reports
+=======
+
+Permanent Inventory Reports: PLE 12.1 and PLE 13.1
+--------------------------------------------------
+
+Two permanent inventory reports can be produced in Odoo for Peruvian accounting:
+PLE 12.1 and PLE 13.1. All inventory transactions made need to be reported. PLE 12.1 only records
+transactions by physical units, while PLE 13.1 tracks both physical units and the monetary value of
+inventory.
+
+Context
+~~~~~~~
+
+PLE 12.1: Permanent Inventory Record in Physical Units
+******************************************************
+
+This record **only tracks inventory in physical units**, focusing on the inflow and outflow of goods
+for effective management and planning. It must be maintained semi-annually (January-June and
+July-December) with monthly transaction details reported within these periods. The submission
+deadlines are October 1st for the first semester and April 1st for the second semester,
+in accordance with Superintendent Resolution 169-2015.
+
+PLE 13.1: Permanent Valued Inventory Record
+*******************************************
+
+This record tracks **both physical quantities and monetary values of inventory**, providing a
+comprehensive view for tax and management purposes. It must be maintained semi-annually
+(January-June and July-December) with monthly transaction details reported within these periods. The
+submission deadlines are October 1st for the first semester and April 1st for the second semester,
+in accordance with Superintendent Resolution 169-2015.
+
+Configuration
+~~~~~~~~~~~~~
+
+Before generating the PLE 12.1 or PLE 13.1 reports, it is recommended to
+:ref:`install and update <general/install>` all Peruvian modules (`l10n_pe`).
+
+Most importantly, install the **Peru - Stock Reports** module (`l10n_pe_reports_stock`).
+
+Update relevant fields
+**********************
+
+Once the **Peru - Stock Reports** module is installed, consider updating and completing the fields
+for:
+
+- :ref:`Products <peru/reports/products>`
+- :ref:`Warehouses <peru/reports/warehouses>`
+- :ref:`Inventory transfers <peru/reports/transfers>`
+
+.. _peru/reports/products:
+
+Products
+^^^^^^^^
+
+There are a few ways to configure a product for PLE reporting:
+
+- **Type of existence:** Navigate to: `Sales --> Product --> Product`. Select a product. In the
+  :guilabel:`Accounting` tab, select the :guilabel:`Type of Existence` according to SUNAT's Table 5
+  for inventory reporting.
```

Comment:
Pull ":" out of bold formatting, use :menuselection: role for nav instructions, and update formatting
```suggestion
- **Type of existence**: Navigate to :menuselection:`Sales --> Products --> Products` and select a
  product. In the :guilabel:`Accounting` tab, select the :guilabel:`Type of Existence` according to
  |SUNAT|'s table 5 for inventory reporting.
```